react/week2/todolist/src/components/Todos.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+
+function AddTodo() {
+  const [todolist, setTodolist] = useState([
+    {
+      id: 1,
+      description: "Get out of bed",
+      completed: true,
+    },
+    {
+      id: 2,
+      description: "Brush teeth",
+      completed: false,
+    },
+    {
+      id: 3,
+      description: "Eat breakfast",
+      completed: false,
+    },
+    {
+        id: 4,
+        description: "Learn React",
+        completed: false,
+      },
+      {
+        id: 5,
+        description: "Finished Web Development",
+        completed: false,
+      }
+  ]);
+  
+  function addTodos() {
```

Comment:
So here since you can name your component Todos, this function should just addTodo (singular). After all, it add one todo, not several.